### DISCUSSION TITLE
Submission details

### DISCUSSION THREAD

**Discussion by IllierZer** [larics/icuas22_competition#48] [Q&A]
@fpetric @lmark1 
We are still a but unsure about how to submit our solution. when we first installed and built the docker the icuas22_competition folder was created so can we just add our code in the scripts directory of that folder, update the session.yml, zip this folder and submit that itself?

**Comment by fpetric** [comment]
Hi @IllierZer, 

I think that will work fine. Just make sure to add launching of your nodes/scripts to the solution pane in the session.yml file, evaluator relies on all of your code being launched from there.
